packages/backend-for-frontend/src/routers/swaggerRouter.ts

```diff
@@ -0,0 +1,31 @@
+import { fileURLToPath } from "url";
+import { readFileSync } from "fs";
+import path from "path";
+import { zodiosRouter } from "@zodios/express";
+import { bffApi } from "pagopa-interop-api-clients";
+import swaggerUi from "swagger-ui-express";
+import YAML from "yaml";
+import { config } from "../config/config.js";
+
+const YML_SPEC_PATH = "../../../api-clients/open-api/bffApi.yml";
```

Comment:
we should check how this works when building to a Docker image – maybe this path should be configurable to be able to set it to something that will make sense once it is built?